internal/util/importutilv2/parquet/field_reader.go

```diff
@@ -94,7 +96,9 @@ func (c *FieldReader) Next(count int64) (any, error) {
 			return nil, nil
 		}
 		return data, typeutil.VerifyFloats64(data.([]float64))
-	case schemapb.DataType_VarChar, schemapb.DataType_String:
+	case schemapb.DataType_VarChar:
```

Comment:
ditto, let's just check both varchar and string!

Reply:
Clubbed them together

Reply:
Also, if you look at the earlier implementation.

ReadStringData() was being used for schemapb.DataType_String and in ReadJSONData() and ReadSparseFloatVectorData().I did not want to add `if else statements` everywhere in the function.So, I created this ReadVarcharData() and now it will be used for both string and varchar datatype. I hope that is fine? @bigsheeper

Reply:
excellent